src/components/ConditionField/styles/ConditionField.css.ts

```diff
@@ -14,19 +11,12 @@ export const FieldCloseWrapperUI = styled(Flexy.Item)`
   margin: 2.5px 0;
 `
 
-export const ConditionFieldUI = styled('div')`
-  ${baseStyles};
-`
+export const ConditionFieldUI = styled('div')``
 
-export const AddButtonWrapperUI = styled('div')`
-  ${baseStyles};
-`
+export const AddButtonWrapperUI = styled('div')``
 
 export const AddButtonContentUI = styled('div')`
-  ${baseStyles};
   padding: 5px 0;
 `
 
-export const OrWrapperUI = styled('div')`
-  ${baseStyles};
-`
+export const OrWrapperUI = styled('div')``
```

Comment:
There are a lot of these empty `styled('div')` components, which seem to be mostly wrappers. Do you think we can remove these now?

Reply:
Nice catch. I went ahead in 8fa835ed and replace all empty styled components